config/options.go

```diff
@@ -33,5 +34,9 @@ func NewCLIOptions(cli map[string]interface{}) (*CLIOptions, error) {
 		},
 	}
 
+	if cli["--partials-dir"] != nil {
+		c.PartialTemplatesPath, _ = cli["--partials-dir"].(string)
+	}
```

Comment:
To stimulate convention over configuration maybe provide a default for the template path?

```
} else {
  // default
  c. PartialTemplatesPath = "config/partials"
```

Reply:
Hmm, not sure, there isn't really a default that I think would fit all use-cases. I still think 99% of the time you don't need partials, since a `charts.yml` is small enough to be less than 50 lines.

At Blendle we use `config/deploy/partials`, but that's not a universal standard or anything. And since our CI sets the `--partials-dir` by default, and locally we use `script/deploy` that abstracts this away, I see no upside to providing a default for this flag.

Feel free to convince me otherwise though 😄

Reply:
You right that in 99% of the times we don't need partial but having a default dir and checking if that exists won't be any overhead performance wise but you will get an overhead for the 1% cases you do want to use partials. Since then you need to lookup what being used by our CI or you configure another path and if you switch projects then you will have to look at yet another path. 

And sure it's not the universal standard but on the other hand there isn't nothing wrong with enforcing standards of our own if it helps us. Remember we first make the tools for ourselves and just make them public available feels much same like a conversation we once had about another open source project. If other people choose to use another path they can still pass along the cli option ;) 

And obviously script/deploy abstracts this away but why should we bother if we have a standard path? I would really like to add a default value, I don't see any negative issues with it, perhaps you have some better arguments to convince me then the once you just mentioned?

Reply:
No, I don't have any better arguments, because I still think my original argument is good enough  😋 

But, I can see your point, and even though I disagree, I am going to make this change before I merge this PR 👍

Reply:
Changed in latest commit 👍